akka-stream/src/main/scala/akka/stream/io/InputStreamSource.scala

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream.io
+
+import java.io.InputStream
+
+import akka.stream.io.impl.InputStreamSource
+import akka.stream.scaladsl.Source
+import akka.stream.scaladsl.Source._
+import akka.stream.{ OperationAttributes, javadsl }
+import akka.util.ByteString
+
+import scala.concurrent.Future
+
+object InputStreamSource {
+
+  final val DefaultChunkSize = 2048
```

Comment:
I was just thinking that most input streams will be BufferedInputStream, and there might be some advantage of using same size (but I'm just guessing)